README.flow-typed.md

```diff
@@ -0,0 +1,15 @@
+# Making Flow Aware of the Types for Our Dependencies
+
+Flow needs help understanding dependencies that do not themselves use Flow. For this purpose, we use `npx flow-typed <package@version>` to create files that describe the types used in a given package. Thus far we have used this for three packages, as outlined in these commands:
```

Comment:
"Thus far we have used this for three packages,...": "three" packages is no longer accurate. Suggestion: "Thus far we have used this for the following packages,...".

Reply:
Good catch, and fixed.